react/styles.css

```diff
@@ -0,0 +1,3 @@
+.popper {
+  z-index: 1300;
+}
```

Comment:
Why not use the `z-max` token from Tachyons?
https://github.com/vtex/vtex-tachyons/blob/master/tachyons.css#L1668

Reply:
> Why not use the `z-max` token from Tachyons?
> https://github.com/vtex/vtex-tachyons/blob/master/tachyons.css#L1668

`z-max` is too big and it should be a hierarchy of elements that will show up in on the screen, because if you imagine a situation where a popover is open and a modal suddenly open. if the popover is `z-max` and the modal is `z-max` too, but the modal uses a portal and the popover don't, it will be shown above the modal which is very weird